packer/build_deb_image.sh

```diff
@@ -292,14 +295,22 @@ export PACKER_LOG_PATH
 if $DRY_RUN ; then
   echo "DryRun: No need to grep errors on log"
 else
+  GREP_STATUS=0
   if [ "$TARGET" = "aws" ]; then
     grep "us-east-1:" $PACKER_LOG_PATH
+    GREP_STATUS=$?
   elif [ "$TARGET" = "gce" ]; then
       grep "A disk image was created" $PACKER_LOG_PATH
+      GREP_STATUS=$?
   elif [ "$TARGET" = "azure" ]; then
     grep "Builds finished. The artifacts of successful builds are:" $PACKER_LOG_PATH
+    GREP_STATUS=$?
+  else
+    echo "No Target is defined"
+    GREP_STATUS=1
   fi
```

Comment:
Instead of those if else, let's make it more clear and rubust. one why to do it:
```
 case "$TARGET" in
      "aws")
        grep "us-east-1:" $PACKER_LOG_PATH
        ;;
      "gce")
        grep "A disk image was created" $PACKER_LOG_PATH
        ;;
      "azure")
       grep "Builds finished. The artifacts of successful builds are:" $PACKER_LOG_PATH
       ;;
      *)
        echo "No Target is defined"
        exit 1
    esac
```